manifests/params.pp

```diff
@@ -5,38 +5,46 @@
 
   case $::osfamily {
     'Debian': {
-      $dhcp_dir    = '/etc/dhcp'
-      $packagename = 'isc-dhcp-server'
-      $servicename = 'isc-dhcp-server'
-      $root_group  = 'root'
-      $bootfiles   = {
+      $dhcp_dir     = '/etc/dhcp'
+      $dhcp_conf6   = 'dhcpd'
+      $packagename  = 'isc-dhcp-server'
+      $servicename  = 'isc-dhcp-server'
+      $servicename6 = 'isc-dhcp-server'
```

Comment:
Does this run the v6 service as well? Otherwise it should be ```undef``` so the validation fails and users know ahead of time it won't work rather than halfway during the process.

Reply:
It gets complicated. In cent7, dhcp4 and dhcp6 are delivered in one package, but run under two independent services dhcpd, dhcpd6). Each services refrences an independent conf file as well.

In debian, dhcp4 and dhcp6 are delivered in the same package and run in the same service. They also share the same conf file. This is my logic behind using $::dhcp::params::dhcp_conf6 as the target for dhcp::dhcp6 concat fragments.

Reply:
It's always more tricky than you think ...